code/frameworks/nextjs/src/next-image-loader-stub.ts

```diff
@@ -36,4 +36,5 @@ const nextImageLoaderStub: RawLoaderDefinition<LoaderOptions> = async function N
 
 nextImageLoaderStub.raw = true;
 
-export = nextImageLoaderStub;
+export default nextImageLoaderStub;
+export const raw = true;
```

Comment:
what is this `raw` export?

Reply:
That's instructing `webpack` to have `content` be a `Buffer` rather than a `string`.
https://webpack.js.org/api/loaders/#raw-loader

Reply:
isn't it the same as line 37 above though, `nextImageLoaderStub.raw = true;`? are both needed?